### DISCUSSION TITLE
learnpytorch.io book website - What is the stack used to create learnpytorch.io book from Github?   Really cool!

### DISCUSSION THREAD

**Discussion by MathewT** [mrdbourke/pytorch-deep-learning#521] [General]
Is there a tutorial on how to create and update the online book like learnpytorch.io?

**Comment by mrdbourke** [comment]
Hi @MathewT ,

There's no tutorial, however, the book is made using the GitHub Action here: https://github.com/mrdbourke/pytorch-deep-learning/blob/main/.github/workflows/make_docs.yml 

In essence, it uses:
1. Material for Mkdocs (creates the documentation), see: https://squidfunk.github.io/mkdocs-material/
2. `mkdocs-jupyter` (converts notebooks to Mkdocs style), see: https://github.com/danielfrg/mkdocs-jupyter

Once a change is made (a git commit) in the notebooks in the `pytorch-deep-learning` directory, the docs get updated and pushed.
